iib/web/api_v1.py

```diff
@@ -90,6 +91,22 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
+    """
+    # Test DB connection
+    try:
+        db.engine.execute(text('SELECT 1'))
+    except Exception:
+        raise IIBError('Database health check failed.')
```

Comment:
Do the details of the underlying exception ever get logged?  If we only pass "Database health check failed" to the caller and no details of the caught exception here are logged anywhere, it doesn't seem debuggable.

I don't see any logging, but it's possible it's somehow handled by app.register_error_handler called elsewhere.

Reply:
I added a log message. Maybe @yashvardhannanavati or @mprahl could comment if it seems appropriate to them.

Reply:
This looks fine to me